magefiles/go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/kubernetes/ingress-nginx/magefiles
+
+go 1.19
+
+require (
+	github.com/blang/semver/v4 v4.0.0
+	github.com/helm/helm v2.17.0+incompatible
+	github.com/magefile/mage v1.14.0
+)
+
+require (
+	github.com/BurntSushi/toml v1.2.1 // indirect
+	github.com/Masterminds/semver v1.5.0 // indirect
+	github.com/cyphar/filepath-securejoin v0.2.3 // indirect
+	github.com/ghodss/yaml v1.0.0 // indirect
+	github.com/gobwas/glob v0.2.3 // indirect
+	github.com/golang/protobuf v1.5.2 // indirect
+	github.com/stretchr/testify v1.8.1 // indirect
+	google.golang.org/protobuf v1.28.0 // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
+	k8s.io/apimachinery v0.25.4 // indirect
+	k8s.io/helm v2.17.0+incompatible // indirect
```

Comment:
Still not much idea about mage @strongjz but I hope there is no impact from using this version of helm. Current helm release os v3.10 so v2.17 is too old.

Reply:
its the templating engine so no worries about our helm